goserver/server/server.go

```diff
@@ -0,0 +1,38 @@
+package server
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+)
+
+const (
+	launchPath = "/launch"
+)
+
+var authorizedFhirURL = flag.String("authorized_fhir_url", "", "The FHIR base url that is authorized to launch the telehealth app. If not set, launch endpoint will always return HTTP 401.")
```

Comment:
Oh sorry I meant just the validation should go here, actual flag definitions should stay in main.

Reply:
Also let's return an error on startup to have a better experience here.

Reply:
Ah got it